Styles/styles.css

```diff
@@ -0,0 +1,233 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+
+/************
+*Site Header*
+*************/
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+  background: white;
+}
+
+header {
+  position: fixed;
+  background: #fff;
+  width: 100%;
+  height: 88px;
+}
+
+
+#logo {
+  float: left;
+  width: 100px;
+  height: auto;
+  padding: 1em 0 0 1em;
+}
+
+header ul {
+  float: right;
+  padding: 4em 2em 0 0;
```

Comment:
I recommend using % for padding and borders and rem for font sizes - this is what Jamie strongly suggested.